test/stage1/behavior/pointers.zig

```diff
@@ -67,10 +67,10 @@ test "C pointer comparison and arithmetic" {
             expect(ptr1 == 0);
             expect(ptr1 >= 0);
             expect(ptr1 <= 0);
-            expect(ptr1 < 1);
-            expect(ptr1 < one);
-            expect(1 > ptr1);
-            expect(one > ptr1);
+            // expect(ptr1 < 1);
```

Comment:
?

Everything else looks good

Reply:
The compiler tries to convert `1` to a pointer to match the LHS but now fails because of the newly-added checks. Don't know what's the right approach here.

Reply:
It's planned to remove the auto casting of integers to C pointers, so I'm fine with this being commented out for now.